docs/update_bios.rst

```diff
@@ -0,0 +1,52 @@
+BIOS Updates
+============
+
+Below are the steps for updating the BIOS on the app and mon servers. You can also follow these steps to update the BIOS on your computer, such as the Admin Workstation Computer.
+
+What you need
+~~~~~~~~~~~~~
+
+  #. A clean USB device to download the ``.bio`` file
+  #. An internet-connected workstation, such as the *Admin Workstation*
+  #. A UPS (uninterrupted power supply), such as a surge-protecting power supply with a backup battery
+  #. A keyboard and mouse to plug into the servers (you can skip this if you are updating a laptop.)
+
+Perform Backups
+~~~~~~~~~~~~~~~
+
+Before performing any updates on the servers, we recommend you :doc:`back up the app server <backup_and_restore>`.
+
+Preparing the USB Stick
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Using the Disks program, delete existing partitions on the USB device, if applicable, and reformat the entire device with one FAT32 partition. You do not need to set a passphrase or encryption on this device. Note that you will lose access to all existing data on this USB stick.
+
+Download and Verify Appropriate .bio Files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  .. note:: We will be following the F7 BIOS update method, `as documented by Intel (available as a PDF)`_.
+
+Check the make and model of your servers and proceed to the Intel Download Centre to download the correct file. Each make and model of Intel NUC will offer various file types; select the ``.bio`` option.
+
+  .. warning:: Do not download BIOS updates from anywhere other than the manufacturer's website. Be sure that you are `on the correct website`_ and that it has a valid SSL Certificate. Intel's SSL Certificate is issued to \*.intel.com and signed by DigiCert. Be sure you download the files specific to the model of your servers.
+
+Intel provides an md5 checksum on the download page. Once you have downloaded the ``.bio file``, using the **Files** application, browse to the file, right click and select **Properties** ▸ **Digests**, select MD5, and click Hash. Compare the result in the Digest column to the md5 sum listed on Intel's website. If these two values do not match, do not proceed, and contact support@freedom.press. Tails `provides a detailed explanation of this process`_.
+
+Copy the .bio file to your USB device using ``dd if=/path/to/bio/file of=/path/to/usb/device``. Be very careful to select the correct path to your USB device, which you can verify by looking in the Disks application. 
```

Comment:
Consistent nomenclature for the Disks utility - it's "application" here, "program" elsewhere.

Reply:
Changed to "application" as consistent with other docs

Reply:
Actually, the docs use both "application" (set_up_transfer_and_export_device.rst, journalist.rst) and "program" (set_up_tails.rst, set_up_admin_tails.rst) etc. This may be a separate small PR to standardize.